server/src/lib.rs

```diff
@@ -266,6 +281,20 @@ impl InitData {
                             openapi::openapi(),
                             swagger_oidc.clone(),
                         ));
+                    svc.service(
```

Comment:
Can we move this endpoint setup to something like `modules/graphql/src/endpoints.rs` matching other modules?

Reply:
Done